spring-rabbit/src/main/java/org/springframework/amqp/rabbit/core/RabbitTemplate.java

```diff
@@ -983,7 +1008,7 @@ else if (isChannelTransacted()) {
 				return RabbitTemplate.this.buildMessageFromResponse(response);
 			}
 			return null;
-		}, obtainTargetConnectionFactory(this.receiveConnectionFactorySelectorExpression, queueName));
+		}, obtainTargetConnectionFactory(this.receiveConnectionFactorySelectorExpression, queueName), false);
```

Comment:
Why is `false` here explicit?

Reply:
Because it's a `receive()` operation, not a publish.